share/spack/gitlab/cloud_pipelines/.gitlab-ci.yml

```diff
@@ -110,6 +152,7 @@ default:
 
 e4s-pr-generate:
   extends: [ ".e4s", ".pr-generate"]
+  tags:
```

Comment:
@eugeneswalker I didn't notice in review that this would override the tag list inherited from the `.generate` job, causing these to run untagged.  Do you recall what the goal was here?